vcf/model.py

```diff
@@ -117,6 +117,22 @@ def is_het(self):
             return None
         return self.gt_type == 1
 
+    @property
+    def is_filt(self):
+        """ Return True for filtered calls """
+        try: # no FT annotation present for this variant
+            FT=self.data.FT
```

Comment:
Could you [PEP 8](https://www.python.org/dev/peps/pep-0008/)-ify this? Preferably no uppercase variable names and spacing around the `=`.